usaspending_api/search/tests/integration/test_spending_over_time.py

```diff
@@ -382,7 +382,7 @@ def _test_correct_response_for_keywords(client):
             {
                 "group": "fiscal_year",
                 "filters": {
-                    "keywords": ["test", "recipient_name_", "recipient_name_1"],
```

Comment:
was there an issue with the trailing underscores?

Reply:
I put that in the original test case to make sure that it wasn't being counted. Now that we are using the wildcard it would trigger every single recipient in the test.

Reply:
ah, thank you